codespell_lib/_codespell.py

```diff
@@ -58,7 +58,7 @@
         True, True, supported_languages_en, supported_languages_en),
     ('usage', 'for replacing phrasing with recommended terms', '_usage',
         None, None, None, None),
-    ('code', 'for words common to code and/or mathematics that might be typos', '_code',  # noqa: E501
+    ('code', 'for words from code and/or mathematics that are likely typos in other context (such as uint)', '_code',  # noqa: E501
```

Comment:
s/context/contexts/

Thanks for this, I'm not sure if the example helps, as it's not necessarily a very strong example, and opens us up to the challenge of our typos appearing in our codebase if we wanted to check with the code dictionary.
```suggestion
    ('code', 'for words from code and/or mathematics that are likely to be typos in other contexts', '_code',  # noqa: E501
```